src/jimgw/samplers/blackjax/nss.py

```
@@ -0,0 +1,190 @@
+            log_likelihood_fn=log_likelihood_fn,
+            log_posterior_fn=log_posterior_fn,
+            config=config,
+        )
+        self._config = config
+        self._stepper_fn = to_prior_space_stepper(config.periodic, parameter_names)
+        self._sampled = False
+
+    def sample(
+        self,
+        rng_key: Key,
+        initial_position: Float[Array, "n_live n_dims"],
```

Ruff (0.15.12)

[error] Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)